test/extended/cli/explain.go

```diff
@@ -54,6 +53,10 @@ var (
 		{Group: "template.openshift.io", Version: "v1", Resource: "templateinstances"},
 	}
 
+	microShiftBuiltinTypes = []schema.GroupVersionResource{
```

Comment:
How hard would it be for us to put these microshift-specific tests in the microshift repo? If not now, eventually?

I know @bparees has been looking at splitting up the tests, so maybe we can be an early adopter of that work?

Reply:
Have local e2e tests? I have seen other operators do it before.